src/homework1/simple_prog16.py

```diff
@@ -0,0 +1,30 @@
+# 18 lines: 8-Queens Problem (recursion)
+
+BOARD_SIZE = 8
+
```

Comment:
here is constant is separated with 2 blank lines (earlier you had 0 line split in task 10 and task14